CHANGELOG.md

```diff
@@ -1,6 +1,10 @@
 Changelog
 =========
 
+v2.0.878 (2021-03-30)
+---------------------
+- Fix: Utilize request sessions for ping identity provider plugin
+
 
```

Comment:
No need to update the changelog, this occurs in an automated process when the team makes a release